sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -1335,7 +1346,7 @@ class PlanResolutionSuite extends AnalysisTest {
     }
   }
 
-  test("MERGE INTO TABLE") {
+  test("MERGE INTO TABLE - primary") {
```

Comment:
This test fails when ANSI is on: https://github.com/apache/spark/runs/6451765082. @dtenedor @gengliangwang

Reply:
I just made a simple followup https://github.com/apache/spark/pull/36570 - seems trivial.